spec/lib/facilities/vha_facility_spec.rb

```diff
@@ -1,89 +1,171 @@
-                                                       'state' => 'PI', 'zip' => '01302')
-          end
+      it 'should include just be 5 digit if zip +4 is empty' do
+        VCR.use_cassette('facilities/va/vha_facilities_limit_results') do
+          expect(facility.address['physical']['zip']).to eq('01302')
+        end
+      end
+
+      it 'should include zip +4 when available' do
+        VCR.use_cassette('facilities/va/vha_facilities_limit_results') do
+          expect(facility_2.address['physical']['zip']).to eq('04330-6796')
         end
+      end
+
+      it 'should include websites for facilities' do
+        VCR.use_cassette('facilities/va/vha_facilities_limit_results') do
+          expect(facility_2.website).to eq('http://www.maine.va.gov/')
+        end
+      end
 
-        it 'should include just be 5 digit if zip +4 is empty' do
+      it 'should include active status for facilities' do
+        VCR.use_cassette('facilities/va/vha_facilities_limit_results') do
+          expect(facility_2.active_status).to eq('A')
+        end
+      end
+
+      it 'should indicate if a facility is mobile' do
+        VCR.use_cassette('facilities/va/vha_facilities_limit_results') do
+          expect(facility.mobile).to eq(false)
+          expect(facility_2.mobile).to eq(true)
+        end
+      end
+
+      it 'should get the correct classification name' do
+        VCR.use_cassette('facilities/va/vha_facilities_limit_results') do
+          expect(facility.classification).to eq('Other Outpatient Services (OOS)')
+          expect(facility_2.classification).to eq('VA Medical Center (VAMC)')
+        end
+      end
+
+      context 'mental health data' do
+        before(:each) do
+          attrs1 = {
+            station_number: '358',
+            mh_phone: '407-123-1234',
+            mh_ext: nil,
+            modified: '2019-09-06T13:00:00.000',
+            local_updated: Time.now.utc.iso8601
+          }
+
+          attrs2 = {
+            station_number: '402',
+            mh_phone: '321-987-6543',
+            mh_ext: '0002',
+            modified: '2019-09-06T13:00:00.000',
+            local_updated: Time.now.utc.iso8601
+          }
+
+          FacilityMentalHealth.create(attrs1)
+          FacilityMentalHealth.create(attrs2)
+        end
+
+        it 'should add mental health info for facilities' do
           VCR.use_cassette('facilities/va/vha_facilities_limit_results') do
-            expect(facility.address['physical']['zip']).to eq('01302')
+            expect(facility.phone['mental_health_clinic']).to eq('407-123-1234')
+            expect(facility_2.phone['mental_health_clinic']).to eq('321-987-6543 x 0002')
           end
         end
+      end
 
-        it 'should include zip +4 when available' do
-          VCR.use_cassette('facilities/va/vha_facilities_limit_results') do
-            expect(facility_2.address['physical']['zip']).to eq('04330-6796')
+      context 'services' do
```

Comment:
When describing a context, start its description with "when" or "with".

```suggestion
  context 'with services' do
```